mlkem/params.h

```diff
@@ -50,12 +53,12 @@
 #define MLKEM_INDCPA_BYTES \
   (MLKEM_POLYVECCOMPRESSEDBYTES_DU + MLKEM_POLYCOMPRESSEDBYTES_DV)
 
-#define MLKEM_PUBLICKEYBYTES (MLKEM_INDCPA_PUBLICKEYBYTES)
+#define MLKEM_INDCCA_PUBLICKEYBYTES (MLKEM_INDCPA_PUBLICKEYBYTES)
 /* 32 bytes of additional space to save H(pk) */
-#define MLKEM_SECRETKEYBYTES                                   \
+#define MLKEM_INDCCA_SECRETKEYBYTES                            \
   (MLKEM_INDCPA_SECRETKEYBYTES + MLKEM_INDCPA_PUBLICKEYBYTES + \
    2 * MLKEM_SYMBYTES)
-#define MLKEM_CIPHERTEXTBYTES (MLKEM_INDCPA_BYTES)
+#define MLKEM_INDCCA_CIPHERTEXTBYTES (MLKEM_INDCPA_BYTES)
```

Comment:
I think this change is undesirable as it requires understanding the difference between CCA and CPA security. 
I'd rather hide the CPA names.

Reply:
Hm, but this is internal?

Reply:
okay